app/code/Magento/SampleData/Console/Command/SampleDataDeployCommand.php

```diff
@@ -137,30 +196,49 @@ private function createAuthFile()
         if (!$directory->isExist(PackagesAuth::PATH_TO_AUTH_FILE)) {
             try {
                 $directory->writeFile(PackagesAuth::PATH_TO_AUTH_FILE, '{}');
-            } catch (\Exception $e) {
-                $message = 'Error in writing Auth file '
-                    . $directory->getAbsolutePath(PackagesAuth::PATH_TO_AUTH_FILE)
-                    . '. Please check permissions for writing.';
-                throw new \Exception($message);
+            } catch (Exception $e) {
+                throw new LocalizedException(__(
+                    'Error in writing Auth file %1. Please check permissions for writing.',
+                    $directory->getAbsolutePath(PackagesAuth::PATH_TO_AUTH_FILE)
+                ));
             }
         }
     }
 
     /**
+     * Updates PHP memory limit
+     *
+     * @throws InvalidArgumentException
      * @return void
      */
     private function updateMemoryLimit()
     {
         if (function_exists('ini_set')) {
-            @ini_set('display_errors', 1);
```

Comment:
Hello @lbajsarowicz 
I removed `@` and added throwing an exception when ini_set returns an error. But we still have the warnings in static tests.
Please check it.
Thank you in advance.